GenderPayGap.WebUI/Views/AdminOrganisationCompaniesHouseOptInOut/OptIn.cshtml

```diff
@@ -91,7 +91,7 @@
                 <tr class="govuk-table__row">
                     <th scope="row" class="govuk-table__header govuk-!-width-one-third">New address</th>
                     @{ OrganisationAddress addressFromCoHo = UpdateFromCompaniesHouseService.CreateOrganisationAddressFromCompaniesHouseAddress(Model.CompaniesHouseCompany.RegisteredOfficeAddress); }
-                    <td class="govuk-table__cell">@(UpdateFromCompaniesHouseService.AddressMatches(addressFromCoHo, Model.Organisation.GetLatestAddress()) ? "(Not changed)" : addressFromCoHo.GetAddressString())</td>
+                    <td class="govuk-table__cell">@(addressFromCoHo.AddressMatches(Model.Organisation.GetLatestAddress()) ? "(Not changed)" : addressFromCoHo.GetAddressString())</td>

```

Comment:
Updated this to use pre-existing version of this function within the class itself!